resources/api/mint-api.yaml

```diff
@@ -333,7 +333,7 @@ definitions:
         items:
           type: string
           example: example-org-stups-mint
-          pattern: "^[a-z0-9][a-z0-9-.]{1,61}[a-z0-9]$"
+          pattern: "^(gs://)?[a-z0-9][a-z0-9-.]{1,61}[a-z0-9]$"
```

Comment:
The pattern works in the swagger editor and it also works when testing locally:

```
$ curl -X PUT -H "Content-Type: application/json" http://localhost:8080/apps/test -d '{"s3_buckets": ["gs://bucket"], "scopes": [], "is_client_confidential": false}' -v
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> PUT /apps/test HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.50.2
> Accept: */*
> Content-Type: application/json
> Content-Length: 78
> 
* upload completely sent off: 78 out of 78 bytes
< HTTP/1.1 200 OK
< Date: Wed, 14 Sep 2016 13:36:38 GMT
< Access-Control-Allow-Origin: *
< Access-Control-Max-Age: 3600
< Access-Control-Allow-Methods: GET, POST, DELETE, PUT, PATCH, OPTIONS
< Access-Control-Allow-Headers: 
< Strict-Transport-Security: max-age=10886400
< Content-Length: 0
< Server: Jetty(9.2.10.v20150310)
< 
* Curl_http_done: called premature == 0
* Connection #0 to host localhost left intact
```

I guess regex101 is using javascript regex where `/` is a special char.